lib/smart_proxy_remote_execution_ssh/multiplexed_ssh_connection.rb

```diff
@@ -60,6 +60,10 @@ def initialize(options, logger:)
       @host_public_key = options.fetch(:host_public_key, nil)
       @verify_host = options.fetch(:verify_host, nil)
       @client_private_key_file = settings.ssh_identity_key_file
+      @client_ca_known_hosts_file = settings.ssh_ca_known_hosts_file
```

Comment:
I'm confused by `ca` in name. This is a path to the known_hosts file where we expect to find a host-pubkey matching, right? What does it have to do with CA?